advanced/Scripts/piholeDebug.sh

```diff
@@ -66,8 +66,8 @@ PIHOLE_DIRECTORY="/etc/pihole"
 PIHOLE_SCRIPTS_DIRECTORY="/opt/pihole"
 BIN_DIRECTORY="/usr/local/bin"
 RUN_DIRECTORY="/run"
-LOG_DIRECTORY="/var/log"
-WEB_SERVER_LOG_DIRECTORY="${LOG_DIRECTORY}/lighttpd"
+LOG_DIRECTORY="/var/log/pihole"
+WEB_SERVER_LOG_DIRECTORY="/var/log/lighttpd"
```

Comment:
I'm thinking that this may be better set up as
`/var/log/lighttpd/pihole/*.log`

Reasons:

We use the following `lighttpd` config:

```conf
server.errorlog             = "/var/log/lighttpd/error.log"
server.pid-file             = "/run/lighttpd.pid"
server.username             = "www-data"
server.groupname            = "www-data"
server.port                 = 80
accesslog.filename          = "/var/log/lighttpd/access.log"
```

If a user has another `lighttpd` SERVER directive on the same node then we may clobber their access and error logs.

Other option to consider would be to use `/var/log/lighttpd/access-pihole.log` or something similar to note that the access and error logs are for Pi-hole.

Reply:
Do you think we need to change `WEB_SERVER_LOG_DIRECTORY`?

Reply:
I would got for

> Other option to consider would be to use /var/log/lighttpd/access-pihole.log or something similar to note that the access and error logs are for Pi-hole.

I'll change the `lighttpd` config

___

> Do you think we need to change WEB_SERVER_LOG_DIRECTORY?

We needed to change it here - it's the debug script that reads the files in the log dir. As we changed it for the Pi-hole files, it's now a different location than for the `lighttpd` logs.

Reply:
Probably worth a dedicated PR, having this one about FTL logs exclusively?

Reply:
PR https://github.com/pi-hole/pi-hole/pull/4747